### ISSUE TITLE
Declarative Project Structures: Scaffolding

### ISSUE THREAD

**Issue by tayfun** [RasaHQ/rasa#10810]
In scope of declarative project structures, add the new `project.yml` file into scaffold template, and remove importers from `config.yml`. 

Make sure `rasa init` creates correct files. Top level project.yml file should have current defaults inside:

```yaml
version: "3.0"

nlu: ['data/nlu.yml']  # array of single files
rules: ['data/rules.yml'] # array of single files
stories: ['data/stories.yml'] # array of single files
config: 'config.yml'  # config is always a single file
domain: ['domain.yml'] # array of file(s) or folder(s)
models: 'models/'  # single file or folder - here single folder means latest model for main-bot used - model name will be generated from bot name
actions: 'actions/' # single folder
# test_data is used for test stories, but our customers can put anything under this folder (cf. NIB)
test_data: ['tests/'] # array of folder(s)
train_test_split: 'train_test_split/' # single folder
results: 'results/' # single folder provided to --out arg

# the following is moved from `config.yml`
# https://rasa.com/docs/rasa/training-data-importers
importers:
    - name: "module.CustomImporter"
      parameter1: "value"
      parameter2: "value2"
    - name: "RasaFileImporter"
```

### Definition of done

- [x] Add project.yml to `rasa/cli/initial_project`
- [x] Make sure `rasa init` creates new project.yml
- [x] Fix any related unit tests
- [x] Unit test the new functionality

**Comment by ancalita** [comment]
@tayfun Two quick q's:
- `rasa/cli/initial_project/config.yml` doesn't have importers specified by default - so there's nothing to be `removed` or did I misread the item in the definition of done?
- considering that the default project doesn't specify importers (if not specified, `RasaFileImporter` gets used, it's somewhere buried in the `TrainingDataImporter` code), I suppose we don't need to add the key in the `project.yml` file in `rasa/cli/initial_project`?

**Comment by tayfun** [comment]
> @tayfun Two quick q's:
> 
> * `rasa/cli/initial_project/config.yml` doesn't have importers specified by default - so there's nothing to be `removed` or did I misread the item in the definition of done?
> * considering that the default project doesn't specify importers (if not specified, `RasaFileImporter` gets used, it's somewhere buried in the `TrainingDataImporter` code), I suppose we don't need to add the key in the `project.yml` file in `rasa/cli/initial_project`?

You are correct on both points. I'll update the ticket.

**Comment by ancalita** [comment]
Closed by #10896
